Week3/hw/app2.js

```diff
@@ -0,0 +1,13 @@
+//  2. Create a function named `colorCar` that receives a color, and prints out, _'a red car'_ for example.
+
+colorCar = 'a red car';
+function colorCar() {
```

Comment:
I don't think this one will work. Because now it will still `console.log`: `a red car`. 
The function `colorCar` doesn't seem to do much